doc/emuwiki-api-doc/Mupen64Plus-v2.0-headers.mediawiki

```diff
@@ -337,3 +337,37 @@
    m64p_error (*VidExtFuncResizeWindow)(int, int);
  } m64p_video_extension_functions;
 
+ /* ------------------------------------------ */
+ /* Structures and Types for Audio Backend API */
+ /* ------------------------------------------ */
+
+ /* Audio backend object
+  *
+  * user_data is a pointer which will ba passed as the first argument of audio backend functions.
+  * Frontend writers are free to use it as they please.
+  *
+  * set_audio_format function allow the audio backend to be notified of the format of incoming samples.
```

Comment:
*The set_audio_format function allows ...* (~~set_audio_format function allow~~), also for `push_audio_samples` below